src/prerna/reactor/frame/FrameToGraphReactor.java

```diff
@@ -0,0 +1,544 @@
+            "  \"data\": [\n" +
+            "    {\n" +
+            "      \"name\": \"placeholder\",\n" +
+            "      \"values\": []\n" +
+            "    }\n" +
+            "  ],\n" +
+            "\n" +
+            "  \"scales\": [\n" +
+            "    {\n" +
+            "      \"name\": \"x\",\n" +
+            "      \"type\": \"point\",\n" +
+            "      \"range\": \"width\",\n" +
+            "      \"domain\": {\"data\": \"table\", \"field\": \"placeholder\"}\n" +
+            "    },\n" +
+            "    {\n" +
+            "      \"name\": \"y\",\n" +
+            "      \"type\": \"linear\",\n" +
+            "      \"range\": \"height\",\n" +
+            "      \"nice\": true,\n" +
+            "      \"zero\": true,\n" +
+            "      \"domain\": {\"data\": \"table\", \"field\": \"placeholder\"}\n" +
+            "    },\n" +
+            "    {\n" +
+            "      \"name\": \"color\",\n" +
+            "      \"type\": \"ordinal\",\n" +
+            "      \"range\": \"category\",\n" +
+            "      \"domain\": {\"data\": \"table\", \"field\": \"c\"}\n" +
+            "    }\n" +
+            "  ],\n" +
+            "\n" +
+            "  \"axes\": [\n" +
+            "    {\"orient\": \"bottom\", \"scale\": \"x\"},\n" +
+            "    {\"orient\": \"left\", \"scale\": \"y\"}\n" +
+            "  ],\n" +
+            "\n" +
+            "  \"marks\": [\n" +
+            "    {\n" +
+            "      \"type\": \"group\",\n" +
+            "      \"from\": {\n" +
+            "        \"facet\": {\n" +
+            "          \"name\": \"series\",\n" +
+            "          \"data\": \"table\",\n" +
+            "          \"groupby\": \"c\"\n" +
+            "        }\n" +
+            "      },\n" +
+            "      \"marks\": [\n" +
+            "        {\n" +
+            "          \"type\": \"line\",\n" +
+            "          \"from\": {\"data\": \"series\"},\n" +
+            "          \"encode\": {\n" +
+            "            \"enter\": {\n" +
+            "              \"x\": {\"scale\": \"x\", \"field\": \"x\"},\n" +
+            "              \"y\": {\"scale\": \"y\", \"field\": \"y\"},\n" +
+            "              \"stroke\": {\"scale\": \"color\", \"field\": \"c\"},\n" +
+            "              \"strokeWidth\": {\"value\": 2}\n" +
+            "            },\n" +
+            "            \"update\": {\n" +
+            "              \"interpolate\": {\"signal\": \"interpolate\"},\n" +
+            "              \"strokeOpacity\": {\"value\": 1}\n" +
+            "            },\n" +
+            "            \"hover\": {\n" +
+            "              \"strokeOpacity\": {\"value\": 0.5}\n" +
+            "            }\n" +
+            "          }\n" +
+            "        }\n" +
+            "      ]\n" +
+            "    }\n" +
+            "  ]\n" +
+            "}";
+    }
+    
+    private String getVegaPieChartTemplate (){
```

Comment:
In future iterations, I would like to move away from building each chart by itself.